src/main/bash/sdkman-install.sh

```diff
@@ -70,7 +70,7 @@ function __sdkman_install_candidate_version() {
 	mkdir -p "${SDKMAN_CANDIDATES_DIR}/${candidate}"
 
 	rm -rf "${SDKMAN_DIR}/tmp/out"
-	unzip -oq "${SDKMAN_DIR}/archives/${candidate}-${version}.zip" -d "${SDKMAN_DIR}/tmp/out"
+	unzip -oq "${SDKMAN_DIR}/tmp/${candidate}-${version}.zip" -d "${SDKMAN_DIR}/tmp/out"
```

Comment:
This is a danger zone :skull:
All binaries will be placed here, including tarballs. As the PR stands now, I think it will break for all tarball distributions. That means the majority of the Java SDKs too.